### ISSUE TITLE
Reports: text-box for date selector does not have an accessible label

### ISSUE THREAD

**Issue by NiquiCara** [rubyforgood/abalone#534]
This textbox does not have a label associated with it.

We can either add a label with `sr-only` or we can add an aria-label tag. I think either will work.

![textbox-without-label](https://user-images.githubusercontent.com/45127691/102560989-8fb75080-40a1-11eb-813e-a2297d7f854b.PNG)

**Comment by CraigJZ** [comment]
Closed by #580
